client/my-sites/customize/main.jsx

```diff
@@ -326,6 +331,18 @@ var Customize = React.createClass( {
 } );
 
 export default connect(
-	( state, props ) => props,
+	( state, props ) => {
+		const siteId = props.sites.getSite( props.domain ).ID;
+		const previousTheme = getCurrentTheme( state, siteId );
+		const queryParams = state.themes.themesList.get( 'query' );
```

Comment:
We should use the `getQueryParams` selector from `state/themes/themes-list/selectors` here (and `queryParams.search` further down, then -- or inline it all here on this line and call the `const` `searchTerm`).